sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/SessionRetryOptionsBuilder.java

```diff
@@ -1,15 +1,12 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did we remove this by mistake? If so, pls put it back.

Reply:
Fixed in next iteration.